front/app/components/admin/PostManager/components/PostTable/header/InitiativesHeaderRow.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+// components
+import { Thead, Tr, Th } from 'components/admin/Table';
+import Checkbox from 'components/UI/Checkbox';
```

Comment:
Could we use the Checkbox from the component library? I want to delete this one at some point..